editors/code/src/commands.ts

```diff
@@ -1517,14 +1517,4 @@ export function openWalkthrough(_: Ctx): Cmd {
             false,
         );
     };
-}
-
-export function openFAQ(_: Ctx): Cmd {
-    return async () => {
-        await vscode.commands.executeCommand(
-            "workbench.action.openWalkthrough",
-            "rust-lang.rust-analyzer#faq",
-            true,
-        );
-    };
-}
+}
```

Comment:
CI probably fails because of the missing newline at the end, please put it back.

Reply:
Thank you for catching that!